src/tracing.rs

```diff
@@ -0,0 +1,29 @@
+use anyhow::Result;
+use tracing_subscriber::filter::LevelFilter;
+use tracing_subscriber::fmt::time::ChronoLocal;
+use tracing_subscriber::FmtSubscriber;
+
+pub fn init(level: LevelFilter, json_format: bool) -> Result<()> {
```

Comment:
I don't see any errors or `?` in this function, why are we returning a `Result` here?